source/agora/cli/checkvtable/main.d

```diff
@@ -0,0 +1,193 @@
+/*******************************************************************************
+
+    Entry point for the agora-gen
```

Comment:
This also. And since it's a single-file program, please add a description of what it does and why.